{"name": "test_login_with_invalid_credentials", "status": "passed", "start": 1695667645104, "stop": 1695667652060, "uuid": "a45fe6c0-d373-4c5b-b292-99e9eba18e45", "historyId": "da1af9640048de1a3dc56211ec8a481c", "testCaseId": "da1af9640048de1a3dc56211ec8a481c", "fullName": "Tests.test_Login.TestPanarayLogin#test_login_with_invalid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestPanarayLogin"}, {"name": "host", "value": "WONIT11L32CV6Z"}, {"name": "thread", "value": "23692-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}